library/ssl_msg.c

```diff
@@ -797,7 +831,18 @@ int mbedtls_ssl_encrypt_buf( mbedtls_ssl_context *ssl,
         /*
          * Encrypt and authenticate
          */
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+        status = psa_aead_encrypt( transform->psa_key_enc,
+                               transform->psa_alg,
+                               iv, transform->ivlen,
+                               add_data, add_data_len,
+                               data, rec->data_len,
+                               data, rec->buf_len - (data - rec->buf),
+                               &rec->data_len );
 
+        if( status != PSA_SUCCESS )
+            return( psa_ssl_status_to_mbedtls( status ) );
```

Comment:
I would suggest adding a similar print, just to make sure we're not losing any debug info:
```suggestion
{
            MBEDTLS_SSL_DEBUG_RET( 1, "psa_aead_encrypt", status );
            return( psa_ssl_status_to_mbedtls( status ) );
}
```